app/src/main/java/com/github/sdpteam15/polyevents/fragments/EventListFragment.kt

```diff
@@ -48,7 +48,7 @@ class ListFragment : Fragment() {
         }
 
```

Comment:
```
 override fun onResume() {
        super.onResume()
        currentDatabase.eventDatabase!!.getEvents(null, null, events).observe(this) {
            if (!it.value) {
                HelperFunctions.showToast("Failed to get events information", context)
            }
        }
    }
```
The event attendees count on the fragment is not updated after subscribing to an event. putting the querry to database in onResume instead of onCreate should do the job fine. 

And I'm not sure why you limited the count to 10. If you put "null" in the number of events, it sends the whole event list.

Reply:
This wasn't my code, but good find for the number of attendees not updated. Will fix this.

As for the limit, I think whoever wrote thought it's be better not to fetch all the events. I will a TODO to see what we can do about this. (Next weekly meeting)